h/api/test/views_test.py

```diff
@@ -65,7 +66,7 @@ def user(monkeypatch):
     return user
 
 
-def test_index():
+def test_index(replace_io):
```

Comment:
Oh and if you're doing this in combination with `@patch`, make sure `@patch` is on the "outside" (i.e. first as you read the file).